src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -83,7 +88,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         private CSharpParseOptions(CSharpParseOptions other) : this(
-            languageVersion: other.LanguageVersion,
+            languageVersion: other.SpecifiedLanguageVersion,
```

Comment:
I think you misunderstood my suggestion. I don't suggest pass effective language version at this position. I suggest to not use constructor that re-computes it, because it has already been computed in the instance we are cloning.

---
In reply to: [74492130](https://github.com/dotnet/roslyn/pull/12969#discussion_r74492130) [](ancestors = 74492130)